docs/src/pages/components/cdk-components/sidenav/Sidenav.jsx

```diff
@@ -28,10 +31,27 @@ function Sidenav() {
           .
         </p>
         <DxcHeading text="DxcSidenav.Title" level={4} weight="bold" />
-        <p>The content will be showed as the main title in the sidenav.</p>
```

Comment:
`DxcSidenav.Title` is not a children of `DxcSidenav`. The prop `title` should be used instead.